ui/app/styles/components/icon.scss

```diff
@@ -47,6 +47,7 @@
 .flight-icon {
   &.flight-icon-display-inline {
     vertical-align: middle;
-    margin: 0px 4px;
+    margin-left: 4px;
```

Comment:
confirmed with Jordan about these styling changes. It fixes the search icon in the search boxes.